tabpy-server/tabpy_server/common/endpoint_file_mgr.py

```diff
@@ -13,25 +13,28 @@
 import shutil
 from re import compile as _compile
 
-logger = logging.getLogger(__name__)
 
 _name_checker = _compile(r'^[a-zA-Z0-9-_\s]+$')
 
 
-def _check_endpoint_name(name):
+def _check_endpoint_name(name, logger=logging.getLogger(__name__)):
     """Checks that the endpoint name is valid by comparing it with an RE and
     checking that it is not reserved."""
     if not isinstance(name, str):
-        log_and_raise("Endpoint name must be a string or unicode", TypeError)
+        msg = 'Endpoint name must be a string or unicode'
```

Comment:
With Py2 being dropped shouldn't reference unicode anymore

Reply:
Yes, we really need to clean up Unicode references everywhere. Not with this change though.